senderscore.py

```diff
@@ -3,25 +3,47 @@
 
 
 class SenderScore(BotPlugin):
-    """Plugin to show the mail score of the given ip"""
+    """Plugin to show the mail score of the given adress"""
 
     @botcmd
     def senderscore(self, msg, args):
-        """retrieve mail reputation usage !senderscore [IP]"""
-        return self.get_senderscores(args)
+        """retrieve mail reputation usage !senderscore [IP/Hostname]"""
+        try:
```

Comment:
the `socket.inet_aton(args)` will raise an exception if its not a IP address. otherwhise it will return a hexadecimal value
i can make an if Boolean for it but then i need to make a separate class for it